.docker/images/govcms8/settings/settings.php

```diff
@@ -145,9 +145,9 @@ public function __toString() {
 if (getenv('LAGOON') && (getenv('ENABLE_REDIS'))) {
   $settings['redis.connection']['interface'] = 'PhpRedis';
   $settings['redis.connection']['host'] = getenv('REDIS_HOST') ?: 'redis';
-  $settings['redis.connection']['port'] = 6379;
+  $settings['redis.connection']['port'] = getenv('REDIS_PORT') ?: 6379;
```

Comment:
Looks good, but Lagoon env var for `REDIS_PORT` includes more than it should (e.g looks to be `REDIS_PORT=tcp://172.30.42.136:6379`)

Reply:
Good catch, thanks! will update to `REDIS_SERVICE_PORT`